frontend/webpack.config.prod.js

```diff
@@ -5,7 +5,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
 const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
 
 module.exports = {
-  devtool: 'source-map',
```

Comment:
Does that work?

Reply:
should help, but isn't enough to speed things up